docs/blog/2018-10-09-vscode-gatsby-development/index.md

```diff
@@ -0,0 +1,88 @@
+
+### ✅ Code linting and Formatting
+
+The main [Gatsby GitHub repo](https://github.com/gatsbyjs/gatsby) has configuration files in place already for [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/). VS Code has extensions for both of these, which makes development really easy.
+
+To get started, go to the Extensions View and [search](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions) for these plugins and press the green Install button for each. You may have to reload Code for the Extensions to take effect.
+
+- Prettier - Code formatter
+- ESLint
+
+Now as you view, create, and modify files ESLint will display "squiggles" for warnings and errors in your files according to the existing rules already configured. You can correct any issues as you go.
+
+For Prettier, you have two options. You can either run the Format manually, using `CMD + Shift + P -> Format Document`. This will format the currently open File.
+
+The other option is to configure Prettier to Format on Save. You can enable the setting `editor.formatOnSave` by [editing the Preferences](https://code.visualstudio.com/docs/getstarted/settings) and applying this at the _User_, _Workspace_, or _Folder_ level.
+
+### Other Fun Plugins
+
+Of course once you start browsing the Extensions Marketplace you'll want to install additional plugins. Here is a short-list of _must-have_ plugins you should consider installing.
+
+- Babel ES6/ES7
+- JSON Tools
+- markdownlint
+- Rainbow Brackets
+- vscode-icons
+
+You can take a look at my ["dotfiles" Github](https://github.com/mikelax/dotfiles/tree/master/vscode) for a README on the VS Code Extensions I am using along with editor settings.
+
+## 🚀 Debugging in Chrome 🚀
+
+VS Code has a built-in debugger to enable proper runtime debugging, if you want to move past the `console.log` throught your code. Use the following steps to be up and (debugging) in minutes.
+
+1. Install the [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) Extension for VS Code.
+1. In the Debug view, click the Debug Dropdown in the Panel and select the option `Add Config (projectname)`. This will create a `launch.json` file in your `.vscode` subfolder in the given project. Add the following to this file.
+
+_NOTE_: You most likley will want to add the `.vscode` folder to your `.gitignore` file so it is not checked into source.
+
+```json5
+// launch.json
+{
+  version: "0.2.0",
+  configurations: [
+    {
+      type: "chrome",
+      request: "launch",
+      name: "Launch Chrome for Gatsby site",
+      url: "http://localhost:8000",
+      webRoot: "${workspaceFolder}",
+    },
+  ],
+}
+```
+
+## Conclusion
+
+In this article we have learned some of the basics about configuring and using VS Code for development on the Gatsby project. The Extensions and configuration discussed here can be applied though to a project that **uses** Gatsby just the same.
```

Comment:
Maybe just say _any_ project, be it a Gatsby project or something else? Largely, these are great VSCode settings for a modern front-end project.